manual/manual_circuit.rst

```diff
@@ -482,6 +482,30 @@ Another notable example that is common to many algorithms and high-level circuit
     circ.add_pauliexpbox(PauliExpBox([Pauli.X, Pauli.Y, Pauli.Y, Pauli.Y], 0.2), [0, 1, 2, 3])
     circ.add_pauliexpbox(PauliExpBox([Pauli.Y, Pauli.X, Pauli.Y, Pauli.Y], -0.2), [0, 1, 2, 3])
 
+Statevectors and unitaries
```

Comment:
It looks like you haven't committed the rebuilt manual: `docs/manual/manual_circuit.html` does not contain this change.